src/main/java/org/datadog/jenkins/plugins/datadog/model/BuildPipelineNode.java

```diff
@@ -313,10 +317,18 @@ public Throwable getErrorObj() {
         return errorObj;
     }
 
+    public String getUnstableMessage() {
+        return unstableMessage;
+    }
+
     public boolean isError() {
         return "error".equalsIgnoreCase(this.result);
     }
 
+    public boolean isUnstable() {
+        return "unstable".equalsIgnoreCase(this.result);
```

Comment:
could we have this precalculated? string equals has more overhead than having a boolean field

Reply:
If we precalculate it there's more room for error (eg: we have to remember to update `isError` and `isUnstable` in `setResult`) and taking into account it's only called 2 times per span, I don't think it's worth the optimization.

I know performance overhead has been sometimes an issue for some customers, but I don't think the impact of this is noticeable (compared to, for example, the call to `hostname` we make once for every span!).